### PR TITLE
GLT-411 Add Plate valid'n (and fix Pool valid'n)

### PR DESCRIPTION
**Pull request by callunity**

### CODE REVIEW

**Comment by apmasell** [comment]
:+1:

File: miso-web/src/main/webapp/styles/style.css [hunk 1]
```diff
@@ -1848,90 +1848,6 @@ button.disabled {
   opacity: .4;
 }
 
- .portlet {
```
Comment (callunity) [inline miso-web/src/main/webapp/styles/style.css]: This CSS styling was introduced in an earlier commit by me, and I am now deliberately removing it.

**Comment by djcooke** [comment]
:+1: other than the 404 on creating new pool

**Comment by callunity** [comment]
Fixed the 404 on creating a new pool

**Pull request merged**